Automattic/wp-calypso
# Hide the traffic breakdown if we cannot display stats

Related to #

## Proposed Changes

- If stats are disabled (when organic_views is -1) then we just display this instead in "traffic breakdown"
 
![image](https://github.com/Automattic/wp-calypso/assets/43957544/bc0dd466-ae59-44ff-8015-4bd6f765219f)
 

## Testing Instructions
- Promote a campaign in jetpack site and disable their stats
- Try to open the new created campaign. You should see this new message  (as described in the image)

## Pre-merge Checklist
